docs/models.txt

```diff
@@ -306,6 +306,7 @@ Interface meeting {
     present_user_ids: (user/is_present_in_meeting_ids)[];
     temporary_user_ids: (user/meeting_id)[];
     guest_ids: (user/guest_meeting_ids)[];
+    user_ids: (Id)[]; // Calculated: All ids from temporary_user_ids, guest_ids and all users assigned to groups.
```

Comment:
@normanjaeckel Should this get a reverse relation? In general: Should calculated fields get a reverse relation (if possible)?

Reply:
I hope not ...

Reply:
I currently also do not see the advantage, so for me it is ok.